visionmetrics/regression/accuracy.py

```diff
@@ -0,0 +1,42 @@
+import torch
+from torchmetrics import Metric
+
+from visionmetrics.common.utils import precision_recall_f1_scalar
+
+
+class MeanAbsoluteErrorF1Score(Metric):
+    """
+    Calculates and returns a discretized version of the MeanAbsoluteError metric.
+    Args:
+        error_threshold: float indicating the exclusive threshold below which the mean absolute error is considered a true positive.
+    """
+    def __init__(self, error_threshold=1.):
+        super().__init__()
+        self.error_threshold = error_threshold
+
+        self.add_state("tp", default=torch.tensor(0.0), dist_reduce_fx="sum")
+        self.add_state("fp", default=torch.tensor(0.0), dist_reduce_fx="sum")
+        self.add_state("fn", default=torch.tensor(0.0), dist_reduce_fx="sum")
+
+    def update(self, predictions, targets):
+        if not torch.is_tensor(predictions):
+            raise ValueError("'predictions' must be a float or integer torch.tensor.")
+        if not torch.is_tensor(targets):
+            raise ValueError("'targets' must be a float or integer torch.tensor.")
+        if predictions.shape != targets.shape:
+            raise ValueError(f"'predictions' and 'targets' must have the same shape; got predictions of shape {list(predictions.shape)} and targets of shape {list(targets.shape)}.")
+        n_obs = targets.numel()
+        tp = torch.sum(torch.where(torch.abs(predictions - targets) < self.error_threshold, 1, 0))
```

Comment:
Would it make more sense to set default value of error_threshold = 0 instead of 1.0. As a new user I may end up using the default metric expecting the usual MAE error. This way one has to explicitly set an error threshold. 

In line 29, we can then use a strict <= comparison. 
Also, maybe add a condition check to ensure error_threshold > 0?